src/components/VideoPreview/VideoPreview.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import styles from './VideoPreview.scss';
+
+const VideoPreview = ({ isCardVideo, showPreview, playVideo, previewImageLink }) =>
```

Comment:
| my opinion |
Same as HoverHint.
You can write an interface for your component.